crates/rpc/rpc-types/src/eth/transaction/receipt.rs

```diff
@@ -9,14 +9,16 @@ pub struct TransactionReceipt {
     /// Transaction Hash.
     pub transaction_hash: Option<H256>,
     /// Index within the block.
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub transaction_index: Option<U256>,
     /// Hash of the block this transaction was included within.
     pub block_hash: Option<H256>,
     /// Number of the block this transaction was included within.
     pub block_number: Option<U256>,
     /// Address of the sender
     pub from: Address,
-    /// Address of the receiver. null when its a contract creation transaction.
+    /// Address of the receiver. None when it's a contract creation transaction.
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub to: Option<Address>,
```

Comment:
I think this should be null for creations, but it could be that some clients differ